Makefile

```diff
@@ -916,6 +957,7 @@ ifeq ($(XZ), 0)
 endif
 	$(RM) $(BINARYNAME).tar
 
+# Builds the binary tarball for releases
 binary: $(BINARYTAR)
 
 binary-upload: binary
```

Comment:
you could note that these -upload tasks are strictly for release builds on release machines only, unless you think that's obvious